bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/action/LightActions.java

```diff
@@ -33,7 +33,7 @@
  * @author Jochen Leopold - Initial contribution
  */
 @Component(scope = ServiceScope.PROTOTYPE, service = LightActions.class)
-@ThingActionsScope(name = "hue")
+@ThingActionsScope(name = "hue-light")
```

Comment:
It is OK to rename this scope. This is for API v1, which we assume is the least used version, so if there are breaking changes on this version, it would only affect a minority of users.